packages/theme/include/colors.ts

```diff
@@ -8,3 +8,9 @@ export const danger = '#c00'
 
 export const transparent = 'transparent'
 export const shade = 'rgba(0, 0, 0, 0.5)'
+
+//action button colors
+export const rootAction = '#e74c3c'
```

Comment:
Pick color names - you can use http://chir.ag/projects/name-that-color/#6195ED to pick approximate names. Never name a color after what you use it for. Then you can't properly re-use it. General theme here - anything that is included by other files needs to be generic when respect to the other files that include it.